src/plot/supstance.js

```diff
@@ -57,7 +57,7 @@ module.exports = function(d3_behavior_drag, d3_event, d3_select, d3_dispatch, ac
       accessor.v(d, value);
       annotationSelection.each(plot.annotation.update(annotation, d3_event().y));
       refresh(g, plot, accessor, x, y, annotationSelection, annotation);
-      dispatch.drag(d);
+      dispatch.drag(d, g);
```

Comment:
This is handy, but it should align to d3 the api, generally passing `(d, i)` as parameters and `this` being the node as documented here https://github.com/mbostock/d3/wiki/Selections#on

> The specified listener is invoked in the same manner as other operator functions, being passed the current datum d and index i, with the this context as the current DOM element.